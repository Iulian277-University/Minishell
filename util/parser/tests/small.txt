pwd > 0$UNU.txt; cd ./subdir; pwd > ../02.txt