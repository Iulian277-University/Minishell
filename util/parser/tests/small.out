> pwd > 0$UNU.txt; cd ./subdir; pwd > ../02.txt
Command successfully read!
command_t (
    op == OP_SEQUENTIAL
    cmd1 (
        command_t (
            op == OP_SEQUENTIAL
            cmd1 (
                command_t (
                    scmd (
                        simple_command_t (
                            verb (
                                'pwd'
                            )
                            out (
                                '0';    expand('UNU');    '.txt'
                            )
                        )
                    )
                )
            )
            cmd2 (
                command_t (
                    scmd (
                        simple_command_t (
                            verb (
                                'cd'
                            )
                            params (
                                './subdir'
                            )
                        )
                    )
                )
            )
        )
    )
    cmd2 (
        command_t (
            scmd (
                simple_command_t (
                    verb (
                        'pwd'
                    )
                    out (
                        '../02.txt'
                    )
                )
            )
        )
    )
)


> End of file!
